static/docs-nav.yaml

```diff
@@ -0,0 +1,42 @@
+site_name: 'Roadie Onboarding docs'
```

Comment:
The static directory is not the right place for this file. Stuff in the static dir is copied into the `public` dir when Gatsby builds so this file will be publically available at https://roadie.io/static/docs-nav.yaml

I would put this in `src`.

Reply:
I put it in content to match where these docs reside